common/src/main/java/com/brandwatch/minibcr/common/repository/mention/MentionRowMapper.java

```diff
@@ -13,9 +13,9 @@
 
     @Override
     public Mention mapRow(ResultSet resultSet, int i) throws SQLException {
-        Mention mention = new Mention();
-        mention.setId(resultSet.getLong("id"));
-        mention.setText(resultSet.getString("text"));
+        long id = resultSet.getLong("id");
+        String text = resultSet.getString("text");
+        Mention mention = new Mention(id, text);
```

Comment:
As you're not using these variables anywhere else, you should be able to return it in one line. Something like:
```suggestion
       new Mention(resultSet.getLong("id"),
                   resultSet.getString("text"));
```
If you added more parameters, a builder pattern or using the setters like you did before would be better.